Client/Application/LeanplumIntegration.swift

```diff
@@ -165,6 +170,9 @@ class LeanplumIntegration {
     // Events
 
     func track(eventName: LeanplumEventName) {
+        guard enabled else {
```

Comment:
shouldSendToLP already checks the enabled bool. Do you still need to check it here again?

Reply:
`shouldSendToLP` checks, but on the main thread. This is, perhaps, an unnecessary optimization.